spec/models/service_template_transformation_plan_spec.rb

```diff
@@ -19,6 +17,81 @@
   let(:flavor1) { FactoryBot.create(:flavor, :name => "large") }
   let(:security_group2) { FactoryBot.create(:security_group, :name => "default") }
   let(:flavor2) { FactoryBot.create(:flavor, :name => "medium") }
+  let(:sg_md) { FactoryBot.create(:security_group, :name => "manageiq-dev") }
+  let(:sg_ch) { FactoryBot.create(:security_group, :name => "converison-host") }
+  let(:sg_cf) { FactoryBot.create(:security_group, :name => "cloudforms") }
```

Comment:
Can you name this something else?  We don't want to use cloudforms in the upstream product